apps/server/src/modules/tool/context-external-tool/service/tool-version-service.ts

```diff
@@ -25,13 +25,25 @@ export class ToolVersionService {
 	): Promise<ToolConfigurationStatus> {
 		// TODO N21-1337 remove if statement, when feature flag is removed
 		if (this.toolFeatures.toolStatusWithoutVersions) {
+			const configurationStatus: ToolConfigurationStatus = new ToolConfigurationStatus({
+				isDisabled: false,
+				isOutdatedOnScopeContext: false,
+				isOutdatedOnScopeSchool: false,
+			});
+
 			try {
```

Comment:
You could try to use a Boolean in the validation functions if it's easy possible